test/integration/targets/eos_eapi/tests/cli/configure.yaml

```diff
@@ -21,13 +21,13 @@
 - name: Expect endpoint ports to be set
   assert:
     that:
-        - http_config.stdout[0].httpServer.port == 81
+        - http_config.stdout[0].httpServer.port == 80
```

Comment:
On line 6 we set the http port to`81`, though we don't enable http
Is the "feature" that you can't set the port without enabling the service, therefore we need to check that the port is set to whatever it was when the service was last enabled?